src/python/pants/backend/codegen/protobuf/python/rules.py

```diff
@@ -94,6 +96,7 @@ async def generate_python_from_protobuf(
             ),
             input_digest=input_digest,
             description=f"Generating Python sources from {request.protocol_target.address}.",
+            level=LogLevel.DEBUG,
```

Comment:
I don't think we should persist this to the user because of the volume of times we run Protoc: one time per target. This is also a fast process.

It will still show up in the dynamic UI.